game-service/src/main/java/org/libertybikes/game/core/OutboundMessage.java

```diff
@@ -39,6 +39,15 @@ public StartingCountdown(int startingSeconds) {
         }
     }
 
+    public static class GameStartCountdown {
```

Comment:
Right now if we did, we would immediately get the spinner for 30 seconds.

It depends on how the frontend wants to handle this and it would be confusing because when we hit 0 we send the other one (which will eventually start the read, set, go) but if the game is full we resend this one with 5 seconds which the frontend couldn't distinguish if they are the same message. The other one should probably be renamed though.